samples/RUNBOOK.md

```diff
@@ -0,0 +1,381 @@
+# Bottlerocket Test System Sample Files
+
+These files contain templates that can be populated using `sed` commands and run with the `cli` tool.
+Examples of how to populate each one of these files can be found below. Each codeblock starts with a small number of variables that need to be populated before running the block, followed by a number of variables that contain default values (but can be modified by the user). `ASSUME_ROLE` has a default value of `~`, but you can replace this with the ARN of an AWS IAM role.
+The final `cat` command will print the populated file to stdout.
+
+## EKS
+
+The files in [eks](./eks) are meant to be run on an EKS test cluster. You can create a new cluster using the [eksctl](https://github.com/weaveworks/eksctl/blob/main/README.md) tool.
+
+### Migration Testing on `aws-ecs` Variants
+
+```bash
+export CLUSTER_NAME=
+
+export VARIANT="aws-ecs-1"
+export ARCHITECTURE="x86_64"
+export AGENT_IMAGE_VERSION=$(cli --version | sed -e "s/^.* //g")
+export ECS_TEST_AGENT_IMAGE_URI="public.ecr.aws/bottlerocket-test-system/ecs-test-agent:v${AGENT_IMAGE_VERSION}"
+export MIGRATION_TEST_AGENT_IMAGE_URI="public.ecr.aws/bottlerocket-test-system/migration-test-agent:v${AGENT_IMAGE_VERSION}"
+export ECS_RESOURCE_AGENT_IMAGE_URI="public.ecr.aws/bottlerocket-test-system/ecs-resource-agent:v${AGENT_IMAGE_VERSION}"
+export EC2_RESOURCE_AGENT_IMAGE_URI="public.ecr.aws/bottlerocket-test-system/ec2-resource-agent:v${AGENT_IMAGE_VERSION}"
+export ASSUME_ROLE="~"
+export AWS_REGION="us-west-2"
+export UPGRADE_VERSION="v1.11.1"
+export STARTING_VERSION="v1.11.0"
```

Comment:
I'm wondering if these should have similar logic to `AGENT_IAMGE_VERSION`

Reply:
It won't be as simple as calling something like `cli --version`. Maybe a `curl` call to `public.ecr.aws/bottlerocket/<agent-image>` and figure out the latest versions that way?

Reply:
Let's save that for a separate pr.